pytorch_lightning/trainer/trainer.py

```diff
@@ -1885,7 +1885,7 @@ def reset_train_dataloader(self, model: Optional["pl.LightningModule"] = None) -
 
         if self.loggers and self.num_training_batches < self.log_every_n_steps:
             rank_zero_warn(
-                f"The number of training samples ({self.num_training_batches}) is smaller than the logging interval"
+                f"The number of training batches ({self.num_training_batches}) is smaller than the logging interval"
```

Comment:
I recall having added a test for this, I hope this did not get lost. Can you check if it's there and update it with the change here?

Reply:
Oh, found it. I made the change directly to your PR. Sorry, I initially thought the PR was by a staff-contributor, hence my direct request.